packages/vue-app/template/utils.js

```diff
@@ -240,6 +241,11 @@ export async function setContext (app, context) {
     }
     if (process.client) {
       app.context.nuxtState = window.<%= globals.context %>
+      // Global IntersectionObserver
+      app.context.observer = {
```

Comment:
If keeping instance in context, why not using factory function like `createObserver`? (this way we can also make util side-effect free)